src/components/sidebar/components/sidebar-filter-input/style.module.css

```diff
@@ -47,8 +47,10 @@
 .filterInput {
   composes: focus-border-and-box-shadow-light from global;
   background-color: transparent;
-  border: 1px solid white;
+  border-color: var(--token-elevation-inset-border-color);
   border-radius: 5px;
+  border-style: solid;
+  border-width: var(--token-elevation-inset-border-width);
```

Comment:
Tweaked styles here to match Figma, which shows border on filter input even on inactive state. Used these `--token-elevation-inset` styles because that seems to be the Effect style in use in Figma.

Reply:
Similar comment, I've updated these styles in my sidebar PR. Could we undo these changes in this PR?